client/src/components/ReviewsCustomerImages.jsx

```diff
@@ -1,42 +1,64 @@
 import React from 'react';
-
+import styled from 'styled-components';
+import ReviewsPopup from './ReviewsPopup.jsx';
 class ReviewsCustomerImages extends React.Component {
 
   constructor(props) {
     super(props);
     this.state = {
-
+      showPopUpModal: false
     };
-
     this.displayAllImages = this.displayAllImages.bind(this);
-
-
   }
 
   displayAllImages(event) {
-
+    this.setState({
+      showPopUpModal: !this.state.showPopUpModal
+    });
   }
 
   render() {
-    const custImageUrls = ['https://homepages.cae.wisc.edu/~ece533/images/pool.png',
-      'https://images.pexels.com/photos/20787/pexels-photo.jpg?auto=compress&cs=tinysrgb&h=350',
-      'https://homepages.cae.wisc.edu/~ece533/images/pool.png',
-      'https://images.pexels.com/photos/20787/pexels-photo.jpg?auto=compress&cs=tinysrgb&h=350',
-    ];
-
+    const ImageContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    padding: 5px;`;
+
+    const Image = styled.img`
+    margin: 5px;
+     width: 153px;
+     height: 153px;
+     `;
+
+    const PopUpModel = styled.div`
+    z-index: auto;
+    display: ${({ show }) => (show ? 'block' : 'none')};
+    position: fixed;
+    top:0;
+    left:0;
+    height:100vh;
+    width: 100vw;
+    background: rgba(0,0,0,0,5);`;
```

Comment:
Consider defining your styled components outside of the render method. The styled-components documentation encourages this practice. [https://styled-components.com/docs/basics#define-styled-components-outside-of-the-render-method](https://styled-components.com/docs/basics#define-styled-components-outside-of-the-render-method)

Reply:
Thank you! I will look into for next PR.